ts/components/icon/SessionIcon.tsx

```diff
@@ -196,6 +197,9 @@ export const SessionIcon = (props: SessionIconProps) => {
   const iconDimensions = getIconDimensionFromIconSize(iconSize);
   const iconDef = icons[iconType];
   const ratio = iconDef?.ratio || 1;
+  const fill = iconDef?.fill || undefined;
+  const clipRule = iconDef?.clipRule || 'nonzero';
```

Comment:
unused currently right? Just future proofing rather than making it forced to nonzero down below?

Reply:
It is used in onboarding for one of the new icons. Just moved that code to here for convenience

Reply:
and I needed the style override prop